impedance/models/circuits/elements.py

```diff
@@ -32,7 +22,14 @@ def wrapper(p, f):
         wrapper.__name__ = func.__name__
         wrapper.__doc__ = func.__doc__
 
+        global circuit_elements
+        if func.__name__ not in circuit_elements:
+            circuit_elements[func.__name__] = wrapper
+
         return wrapper
+
+    # add this circuit to the global list of circuit elements if not already in there
```

Comment:
Can you bump this up to where you're adding it to the global circuit elements variable (looks like line 26)? Also looks like this comment needs to get broken up into a few lines to avoid flake8 errors for a long line